example/src/Examples/WithBottomSheet.tsx

```diff
@@ -0,0 +1,144 @@
+import BottomSheet, {
+  BottomSheetScrollView,
+  BottomSheetView,
+} from '@gorhom/bottom-sheet';
+import { Portal } from '@gorhom/portal';
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React, { useEffect, useState } from 'react';
+import {
+  SafeAreaView,
+  Platform,
+  StyleSheet,
+  Button,
+  Keyboard,
+  type KeyboardEvent,
+  TouchableOpacity,
+  Text,
+  KeyboardAvoidingView,
+} from 'react-native';
+import { RichText, Toolbar, useEditorBridge } from '@10play/tentap-editor';
+
+const useKeyboard = () => {
+  const [keyboardHeight, setKeyboardHeight] = useState(0);
+
+  useEffect(() => {
+    function onKeyboardDidShow(e: KeyboardEvent) {
+      // Remove type here if not using TypeScript
+      setKeyboardHeight(e.endCoordinates.height);
+    }
+
+    function onKeyboardDidHide() {
+      setKeyboardHeight(0);
+    }
+
+    const showSubscription = Keyboard.addListener(
+      'keyboardDidShow',
+      onKeyboardDidShow
+    );
+    const hideSubscription = Keyboard.addListener(
+      'keyboardDidHide',
+      onKeyboardDidHide
+    );
+    return () => {
+      showSubscription.remove();
+      hideSubscription.remove();
+    };
+  }, []);
+
+  return keyboardHeight;
+};
+
+export const WithBottomSheet = ({}: NativeStackScreenProps<any, any, any>) => {
```

Comment:
I am still unable to scroll on Android - seems like the scroll is getting handled by the drawer

https://github.com/user-attachments/assets/613de9e6-f7a4-4d4f-8636-13ec13c1788b